src/libs/actions/Policy.ts

```diff
@@ -250,6 +250,10 @@ function updateLastAccessedWorkspace(policyID: OnyxEntry<string>) {
     Onyx.set(ONYXKEYS.LAST_ACCESSED_WORKSPACE_POLICY_ID, policyID);
 }
 
+function isCurrencySupportedForDirectReimbursement(currency: string) {
+    return CONST.DIRECT_REIMBURSEMENT_CURRENCIES.filter((c) => c !== CONST.CURRENCY.EUR).includes(currency as (typeof CONST.DIRECT_REIMBURSEMENT_CURRENCIES)[number]);
```

Comment:
Based on [this comment](https://github.com/Expensify/App/pull/38318#issuecomment-2007406894), I wonder if we should just check for USD for now if it's true that we only support USD. (I'm not sure why the CONST is wrong though!).

Reply:
cc @luacmartins should we just check for USD currency? 

Yes, it's odd that the CONST is wrong I do agree. In the backend, direct reimbursement is available for `EUR` currency.

Reply:
Yes, we only support USD in NewDot for now. The CONST was added as part of [this issue](https://github.com/Expensify/App/issues/33716) for backward compatibility and is currently behind the global reimbursements beta. We're gonna build that functionality into NewDot as part of a wave-collect or control initiative.